### DISCUSSION TITLE
https://klp1.com/issues/missing-macros/

### DISCUSSION THREAD

**Discussion by giscus** [blakadder/klp1#10] [Announcements]
# https://klp1.com/issues/missing-macros/

Add missing load/unload macros to Kingroon KLP1

https://klp1.com/issues/missing-macros/

**Comment by razvannicu** [comment]
It is possible to add a beeper after succesfuly macro run?

**Reply by AxeOfKukujevo** [reply]
-if not already there
[output_pin beeper]
pin: PC5

then add and edit to your liking
[gcode_macro BEEP]
gcode:  
        SET_PIN PIN=beeper VALUE={1.0}
        G4 P{500}
        SET_PIN PIN=beeper VALUE=0

**Comment by jaymcd** [comment]
Anyone know how to change the Front LED color ( KingRoon KLP1 light) and the LED on the print head? The mini - led color setting doesnt seem to do anything....

**Comment by karabas2011** [comment]
Comment M84 out if you plan to change filament during print to prevent homing
